open_event/forms/admin/microlocation_form.py

```diff
@@ -0,0 +1,18 @@
+"""Written by - Rafal Kowalski"""
```

Comment:
Incorrect way of writing author and License info. See https://code.google.com/p/soc/wiki/PythonStyleGuide#Copyright_And_License_Notices